packages/valory/skills/abstract_round_abci/behaviour_utils.py

```diff
@@ -395,10 +395,9 @@ def __init__(self, **kwargs: Any):
         """Initialize an `IPFSBehaviour`."""
         super().__init__(**kwargs)
         self.ipfs_enabled = False
-        # If params are not found `AttributeError` will be raised. This is fine, because something will have gone wrong.
         # If `ipfs_domain_name` is not specified for the skill, then we get a `None` default.
         # Therefore, `IPFSBehaviour` will be disabled.
-        domain = getattr(self.params, "ipfs_domain_name", None)  # pylint: disable=E1101
+        domain = self.context.params.ipfs_domain_name
```

Comment:
This is now a standard attribute defined on `BaseParams`

Reply:
It will be shifted to the `IPFS` connection soon.